python/tk_multi_launchapp/base_launcher.py

```diff
@@ -12,9 +12,9 @@
 import sys
 import contextlib
 
-if sys.version_info[0:2] >= (3, 10):
```

Comment:
Not sure I want to change that. It's not about Py 2 and we do still support 3.7 and 3.9